test/e2e/performanceprofile/functests/1_performance/cpu_management.go

```diff
@@ -654,6 +528,129 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", Ordered, func() {
 		)
 
 	})
+	Context("Crio Annotations", func() {
+		var testpod *corev1.Pod
+		var allTestpods map[types.UID]*corev1.Pod
+		annotations := map[string]string{
+			"cpu-load-balancing.crio.io": "disable",
+			"cpu-quota.crio.io":          "disable",
+		}
+		BeforeAll(func() {
+			var err error
+			allTestpods = make(map[types.UID]*corev1.Pod)
+
+			testpod = getTestPodWithAnnotations(annotations, smtLevel)
+			// workaround for https://github.com/kubernetes/kubernetes/issues/107074
+			// until https://github.com/kubernetes/kubernetes/pull/120661 lands
+			unblockerPod := pods.GetTestPod() // any non-GU pod will suffice ...
+			unblockerPod.Namespace = testutils.NamespaceTesting
+			unblockerPod.Spec.NodeSelector = map[string]string{testutils.LabelHostname: workerRTNode.Name}
+			err = testclient.Client.Create(context.TODO(), unblockerPod)
+			Expect(err).ToNot(HaveOccurred())
+			allTestpods[unblockerPod.UID] = unblockerPod
+			time.Sleep(30 * time.Second) // let cpumanager reconcile loop catch up
+
+			// It's possible that when this test runs the value of
+			// defaultCpuNotInSchedulingDomains is empty if no gu pods are running
+			defaultCpuNotInSchedulingDomains, err := getCPUswithLoadBalanceDisabled(ctx, workerRTNode)
+			Expect(err).ToNot(HaveOccurred(), "Unable to fetch scheduling domains")
+			if len(defaultCpuNotInSchedulingDomains) > 0 {
+				pods, err := pods.GetPodsOnNode(context.TODO(), workerRTNode.Name)
+				if err != nil {
+					testlog.Warningf("Warning cannot list pods on %q: %v", workerRTNode.Name, err)
+				} else {
+					testlog.Infof("pods on %q BEGIN", workerRTNode.Name)
+					for _, pod := range pods {
+						testlog.Infof("- %s/%s %s", pod.Namespace, pod.Name, pod.UID)
+					}
+					testlog.Infof("pods on %q END", workerRTNode.Name)
+				}
+				Expect(defaultCpuNotInSchedulingDomains).To(BeEmpty(), "the test expects all CPUs within a scheduling domain when starting")
+			}
+
+			By("Starting the pod")
+			testpod.Spec.NodeSelector = testutils.NodeSelectorLabels
+			testpod.Spec.Containers[0].Image = "quay.io/mniranja/busycpus"
```

Comment:
we should probably replace this quay url

Reply:
@yanirq fixed it, using http://quay.io/ocp-edge-qe/busycpus